graph/db/sql_store.go

```diff
@@ -4182,8 +4171,12 @@ func buildChanPolicy(dbPolicy sqlc.ChannelPolicy, channelID uint64,
 		LastUpdate: time.Unix(
 			dbPolicy.LastUpdate.Int64, 0,
 		),
-		MessageFlags:  msgFlags,
-		ChannelFlags:  chanFlags,
+		MessageFlags: lnwire.ChanUpdateMsgFlags(
```

Comment:
nit: could be `ExtractSqlInt16[lnwire.ChanUpdateMsgFlags](dbPolicy.MessageFlags)` instead, if we added the helper as we have them here: https://github.com/lightninglabs/taproot-assets/blob/main/tapdb/sqlutils.go#L124

Reply:
oh cool! added - thanks